apps/explorer/src/components/OwnedCoins/CoinIcon.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useCoinMetadata } from '@mysten/core';
```

Comment:
Can we use this component or expand this? https://github.com/MystenLabs/sui/blob/main/apps/explorer/src/ui/CoinsStack.tsx

Reply:
they've got some decently different styling so I think let's leave this in here